lib/widgets/verse_card.dart

```diff
@@ -32,34 +32,6 @@ class VerseCard extends StatelessWidget {
           if (_header != null)
             Stack(
               children: <Widget>[
-                Align(
```

Comment:
Are we sure we want to remove this "Powered by YouVersion" part? I'm not sure if it's required, but I'd like to give credit to YouVersion if/where possible. Did you remove it because of the look or . . .?

Reply:
I was thinking we could put that into the "About this Page" (Which I still need a design for). What do you think?

Reply:
Yeah okay, I think that'd work, we could put something like "Daily Verses are from YouVersion" or something there (and also reference any other apis we use).